public/app/percona/shared/components/hooks/recurringCall.hook.test.tsx

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { useRecurringCall } from './recurringCall.hook';
+
+let fakeCallback: jest.Mock;
+const runAllPromises = () => new Promise(resolve => setImmediate(resolve));
+
+const Dummy = () => {
+  const [triggerTimeout, changeInterval, stopTimeout] = useRecurringCall();
+
+  return (
+    <>
+      <button onClick={() => triggerTimeout(fakeCallback, 5000, true)}></button>
```

Comment:
Better to use a constant for `5000` and replace the number usage with the constant to make it easier to change the tests if needed.

Reply:
👍